src/components/ContentWidgets/Separator/SeparatorEditor.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import { WidgetEditorProps } from "../../ContentMapping/ContentMapping";
+import { Separator } from "./Separator";
+
+/**
+ * SeparatorEditor edits the width and color of a separator widget
+ *
+ * Last Modified
+ * Jennifer Tao
+ * July 10, 2019
+ */
+export const SeparatorEditor: React.FC<WidgetEditorProps> = ({
+  editedContent,
+  setEditedContentOnChange
+}) => {
+  const defaultColor = { r: 66, g: 13, b: 171 };
+  const width = editedContent.separator_width || 33;
+  const color = editedContent.separator_color || defaultColor;
+
+  /**
+   * Updates seperator_color
+   * @param e event
+   * @param c color: {red, green, blue}
+   */
+  let changeColor = (e: any, c: string) => {
+    let newColor = { ...color };
+    if (c === "red") {
+      newColor.r = e.target.value;
+    } else if (c === "green") {
+      newColor.g = e.target.value;
+    } else if (c === "blue") {
+      newColor.b = e.target.value;
+    }
+
+    setEditedContentOnChange("separator_color", newColor);
+  };
+
+  return (
+    <>
+      <Separator separator_width={width} separator_color={color} />
+      <h3>Size:</h3>
+      Width:
+      <input
+        className="slider width-range"
+        type="range"
+        min="0"
+        max="100"
+        defaultValue={width + ""}
+        onChange={e =>
+          setEditedContentOnChange("separator_width", Number(e.target.value))
+        }
+      />
+      {width}% (Default 33%)
+      <h3>Color:</h3>
+      <input
+        type="button"
+        value="Default Color"
+        onClick={() => {
+          setEditedContentOnChange("separator_color", defaultColor);
+        }}
+      />
+      Default: R: 66, G: 13, B: 171
```

Comment:
I think this is bad user experience, and instead we should have some sort of button that sets the default values or have some presets the user can choose from.